### PR TITLE
update tune-tikv for raftdb

### PR DESCRIPTION
**Pull request by javaforfun**

### CODE REVIEW

File: op-guide/tune-TiKV.md [hunk 1]
```diff
@@ -7,13 +7,18 @@ category: advanced
 This document describes how to tune the TiKV parameters for optimal performance.
 
 TiKV uses RocksDB for persistent storage at the bottom level of the TiKV architecture. Therefore, many of the performance parameters are related to RocksDB.
+Tikv uses two RocksDB instance, the default RocksDB instance store KV data, the Raft RocksDB instance (RaftDB) store Raft logs.
```
Comment (andelf) [inline op-guide/tune-TiKV.md]: use consistent name for TiKV
use period (not comma).
stores

File: op-guide/tune-TiKV.md [hunk 2]
```diff
@@ -7,13 +7,18 @@ category: advanced
...
 
-TiKV implements `Column Families` (CF) from RocksDB and the data is ultimately stored in the `raft`, `default`, `lock` and `write` CFs inside RocksDB.
-+ The `raft` CF stores the Raft log. The corresponding parameters are in `[rocksdb.raftcf]`.
+TiKV implements `Column Families` (CF) from RocksDB.
+
+Default RocksDB store KV data in the `default`, `write` and `lock` CFs.
```
Comment (andelf) [inline op-guide/tune-TiKV.md]: stores

File: op-guide/tune-TiKV.md [hunk 3]
```diff
@@ -7,13 +7,18 @@ category: advanced
 This document describes how to tune the TiKV parameters for optimal performance.
 
 TiKV uses RocksDB for persistent storage at the bottom level of the TiKV architecture. Therefore, many of the performance parameters are related to RocksDB.
+TiKV uses two RocksDB instance: the default RocksDB instance stores KV data, the Raft RocksDB instance (RaftDB) stores Raft logs.
```
Comment (queenypingcap) [inline op-guide/tune-TiKV.md]: two RocksDB instance -> two RocksDB instances

File: op-guide/tune-TiKV.md [hunk 4]
```diff
@@ -7,13 +7,18 @@ category: advanced
...
 
-TiKV implements `Column Families` (CF) from RocksDB and the data is ultimately stored in the `raft`, `default`, `lock` and `write` CFs inside RocksDB.
-+ The `raft` CF stores the Raft log. The corresponding parameters are in `[rocksdb.raftcf]`.
+TiKV implements `Column Families` (CF) from RocksDB.
+
+Default RocksDB stores KV data in the `default`, `write` and `lock` CFs.
```
Comment (queenypingcap) [inline op-guide/tune-TiKV.md]: Default RocksDB stores -> The default RocksDB instance stores

File: op-guide/tune-TiKV.md [hunk 5]
```diff
@@ -7,16 +7,16 @@ category: advanced
 This document describes how to tune the TiKV parameters for optimal performance.
 
 TiKV uses RocksDB for persistent storage at the bottom level of the TiKV architecture. Therefore, many of the performance parameters are related to RocksDB.
-TiKV uses two RocksDB instance: the default RocksDB instance stores KV data, the Raft RocksDB instance (RaftDB) stores Raft logs.
+TiKV uses two RocksDB instances: the default RocksDB instance stores KV data, the Raft RocksDB instance (RaftDB) stores Raft logs.
 
 TiKV implements `Column Families` (CF) from RocksDB.
 
-Default RocksDB stores KV data in the `default`, `write` and `lock` CFs.
+The default RocksDB stores KV data in the `default`, `write` and `lock` CFs.
```
Comment (queenypingcap) [inline op-guide/tune-TiKV.md]: The default RocksDB ->The default RocksDB instance

File: op-guide/tune-TiKV.md [hunk 6]
```diff
@@ -7,16 +7,16 @@ category: advanced
...
 + The `default` CF stores the actual data. The corresponding parameters are in  `[rocksdb.defaultcf]`.
 + The `write` CF stores the version information in Multi-Version Concurrency Control (MVCC). The corresponding parameters are in `[rocksdb.writecf]`.
 + The `lock` CF stores the lock information. The system uses the default parameters.
 
-Raft RocksDB (RaftDB) stores Raft logs.
+The Raft RocksDB (RaftDB) stores Raft logs.
```
Comment (queenypingcap) [inline op-guide/tune-TiKV.md]: The Raft RocksDB (RaftDB) -> The Raft RocksDB (RaftDB) instance

**Pull request merged**